sites/partners/src/components/listings/PaperListingDetails/sections/DetailNeighborhoodAmenities.tsx

```diff
@@ -1,83 +1,95 @@
-import React, { useContext } from "react"
+import React, { useContext, useMemo } from "react"
 import { t } from "@bloom-housing/ui-components"
 import { FieldValue, Grid } from "@bloom-housing/ui-seeds"
 import { ListingContext } from "../../ListingContext"
 import { getDetailFieldString } from "./helpers"
 import SectionWithGrid from "../../../shared/SectionWithGrid"
 import { AuthContext } from "@bloom-housing/shared-helpers"
-import { FeatureFlagEnum } from "@bloom-housing/shared-helpers/src/types/backend-swagger"
+import {
+  FeatureFlagEnum,
+  NeighborhoodAmenitiesEnum,
+  Listing,
+} from "@bloom-housing/shared-helpers/src/types/backend-swagger"
+import { useJurisdiction } from "../../../../lib/hooks"
+
+type AmenityConfig = {
```

Comment:
This is a nit, but I don't think you need this & can use the enum value inline - will add some comments below but this is not blocking!